src/content/docs/journey-large-logs/get-started.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: How to manage large log ingest 
+metaDescription: "Test"
+---
+
+import logsParsing from 'images/logs_screenshot_full-parsing.webp'
+import logsPartition from 'images/logs_screenshot_full-partition.webp'
+
+
+Modern systems create large amounts of log data. You might be dealing with hundreds of gigabytes to dozens of terabytes today, and the amount will continue to increase as your system scales. Anything past a few gigabytes of logs means hours of toil while you search through logs when need arises. Log management solutions, like New Relic, provide the tools to handle large sets of logs and make them managable — and more important, valuable. 
```

Comment:
> Anything past a few gigabytes of logs means hours of toil while you search through logs when need arises.

This is the key point but this sentence feels like it should get sharper. 

I think it's also wroth mentioning cost in here. For a number of o11y vendors, lots of logs = lots of $

Reply:
> Log management solutions, like New Relic, provide the tools...

This makes it sound like what you're about to walk thru is a commoditized, basic feature. Suggest rephrasing to emphasize our unique value prop